processor/common/config.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+// MatchProperties specifies the set of properties in a span to match against
+// and if the span should be included or excluded from the processor.
+// At least one of services, span names or attributes must be specified. It is
+// supported to have all specified, but this requires all of the properties to
+// match for the inclusion/exclusion to occur.
+// The following are examples of invalid configurations:
+//  attributes/bad1:
+//    # This is invalid because include is specified with neither services or
+//    # attributes.
+//    include:
+//    actions: ...
+//
+//  span/bad2:
+//    exclude:
+//    	# This is invalid because services, span_names and attributes have empty values.
+//      services:
+//      span_names:
+//      attributes:
+//    actions: ...
+// Please refer to attributesprocessor/testdata/config.yaml and spanprocessor/testdata/config.yaml for valid configurations.
+type MatchProperties struct {
```

Comment:
```suggestion
type SpanMatchConfig struct {
```
to make it clear this is a config.

Reply:
Moved to internal/processor/span and the type is now MatchConfig to contain both the include and exclude.